### PR TITLE
feat(ui): add storage table to cloning form

### PR DESCRIPTION
**Pull request by Caleb-Ellis**
## Done

- Added machine storage table to cloning form

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Select a couple of machines and open the clone form
- Select a source machine and check that placeholder content shows in the storage table while the machine details are loading
- Check that once the details are loaded the storage table looks like [the design](https://app.zeplin.io/project/60e32f0e5c4c0f13e711d590/screen/6107cdc207416a12bf9794f8)

## Fixes

Fixes canonical-web-and-design/app-squad#195

### CODE REVIEW

File: ui/src/app/machines/views/MachineDetails/MachineStorage/AvailableStorageTable/AvailableStorageTable.tsx [hunk 1]
```diff
@@ -176,6 +175,7 @@ const normaliseRowData = (
                 "—"
               )
             }
+            primaryClassName="u-align--center"
```
Comment (Caleb-Ellis) [inline ui/src/app/machines/views/MachineDetails/MachineStorage/AvailableStorageTable/AvailableStorageTable.tsx:178]: Driveby fix for the machine details storage tab

**Comment by huwshimi** [comment]
Maybe you have another task to work on making it responsive, but I noticed that it switches layout at a really large screen size:

Screen Shot 2021-08-12 at 9 33 35 am

**Review by huwshimi — APPROVED** [review]

**Comment by Caleb-Ellis** [comment]
@huwshimi The full screen (and only) design has squeezed in a crapton of data and it can't be made any smaller without really truncating all the table contents. So I actually made it change layout at that viewport width on purpose, but admittedly at the breakpoint it looks pretty wonky. 😬 What if I made the source machine select half-width so at least all the cards are the same size?

**Comment by huwshimi** [comment]
> @huwshimi The full screen (and only) design has squeezed in a crapton of data and it can't be made any smaller without really truncating all the table contents. So I actually made it change layout at that viewport width on purpose, but admittedly at the breakpoint it looks pretty wonky. 😬 What if I made the source machine select half-width so at least all the cards are the same size?

Yeah I think that might be better than the giant card.

**Comment by Caleb-Ellis** [comment]
@huwshimi cool I'll add a card to improve responsiveness :+1:
